### PR TITLE
Fix every screen not working and wirelinks being broken

### PR DESCRIPTION
**Pull request by NovaAstral**
- Fixed every screen entirely not working because wire/gpulib is being used by every screen
- Fixed wirelinks (placing egp and any e2 using a wirelink input) being broken, not sure exactly why but removing those lines fixes it

### CODE REVIEW

**Comment by yogwoggf** [comment]
@Vurv78 @thegrb93 Could neither of you just test wiremod on the branch Vurv was trying to merge? Actually amazing how bad this repo is 😓

File: lua/wire/server/wirelib.lua [hunk 1]
```diff
@@ -254,13 +254,6 @@ function WireLib.AdjustSpecialOutputs(ent, names, types, descs)
 
 	local ent_ports = ent.Outputs or {}
 
-	if ent_ports.wirelink then
```
Comment (Vurv78) [inline lua/wire/server/wirelib.lua:257]: what is this change

**Review by Vurv78 — CHANGES_REQUESTED** [review]

File: lua/wire/server/wirelib.lua (hunk 1 above)
Comment (Vurv78) [reply lua/wire/server/wirelib.lua:257]: this reverts https://github.com/wiremod/wire/pull/2314. I'm pushing the gpulib removal undo without it. If you can reproduce this breaking anything we can continue the discussion here.

**Comment by yogwoggf** [comment]
Why exactly was my comment removed?

**Pull request closed without merging**